solutions/02_helloName/helloName.js

```diff
@@ -0,0 +1,2 @@
+var name = prompt ("What's your name?");
```

Comment:
nice! just one extra space in between `prompt` and the `(`. It will still work, but just harder to read